client/src/ui/molecules/PersonPod.tsx

```diff
@@ -29,15 +41,24 @@ const PersonPod = ({ initialySelected, toggleHandler, institution, name, tags =
             buttonText="Add"
             onClick={onClick}
         >
+            <Modal
+                hide={toggle}
+                isShowing={isShowing}
+                onAccept={onClick}
+                buttonType="primary"
+                buttonText={`${selected ? 'Remove' : 'Add'} Editor`}
+            >
+                <PersonInfo name={name} institution={institution} focuses={focuses} expertises={expertises} />
+            </Modal>
             <div className="person-pod__text">
                 <span className="typography__body typography__body--primary typography__body--no-margin">{name}</span>
                 <span className="typography__small typography__small--primary typography__small--no-margin">
-                    { institution }
+                    {institution}
                 </span>
                 <div className="person-pod__inline_text">
-                    <Info fontSize="small" />
+                    <Info fontSize="small" onClick={(): void => toggle()} />
```

Comment:
This could do with cursor styling so it's clear it's clickable. Also we should add aria labels